Runtime/Resources/UIWidgets_canvas_strokeAlpha.shader

```diff
@@ -0,0 +1,65 @@
+Shader "UIWidgets/canvas_strokeAlpha"
+{
+    Properties {
+       _SrcBlend("_SrcBlend", Int) = 1 // One
```

Comment:
these two blendmode variables have no reference in this shader, remove them or fix line 12 a bit ?